app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -39,13 +58,30 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
               label="Title"
               name="title"
               className="flex-grow-0 w-full"
+              placeholder="Enter note title"
               required
             />
             <Textarea
               label="Description"
               name="description"
               className="flex-grow-0 w-full"
-              rows={8}
+              placeholder="Enter note description"
+              required
```

Comment:
Do we need `required` here now that we are using Formik? 
Also, please replace all other occurrences.